### PR TITLE
Issue/56 projects page

### PR DESCRIPTION
**Pull request by vallo87**
#56 saving different projects, switching between projects, login with current project reference

### CODE REVIEW

**Comment by stefanorosanelli** [comment]
Seems ok, good job! 

Just a few notes:

-  if I'm not setting a `title` project name is used in parenthesis after login and that's ok but
   + after logout also project name disappears and we get `(not available)` as title
   + there's no way to add or change user `title` after a project is set - you may however remove and create it again (we could do this later)

- message `Confirm switch .. You will be logged out ....` is displayed even if we are not logged into any project

File: src/controllers/pages.js [hunk 1]
```diff
@@ -2,21 +2,55 @@ import { Controller } from '@chialab/synapse';
 import { mix } from '@dnajs/idom';
 import { NotifyMixin } from '../mixins/notify-mixin.js';
 
+
+/**
+ *
```
Comment (stefanorosanelli) [inline src/controllers/pages.js]: missing doc comment?

File: src/controllers/pages.js [hunk 2]
```diff
@@ -2,21 +2,55 @@ import { Controller } from '@chialab/synapse';
...
+                if (err.error.status === '403') {
+                    this.factory('notifications').notify({ body: err.error.title, type: 'error', timeout: 5000 });
+                }
+            }
+        }
+        return this.redirect('/projects');
     }
 
+    /**
+     *
```
Comment (stefanorosanelli) [inline src/controllers/pages.js]: missing doc?

**Comment by stefanorosanelli** [comment]
Everything ok now but the footer refresh problem in #55 has reappeared
You need to refresh the page after login / logout actions

**Comment by vallo87** [comment]
api infos are now coherent, but handling of project/api info in be-footer (also in general) needs a refactor (same api call in multiple places)

**Comment by stefanorosanelli** [comment]
@vallo87 would you open a `be-footer` refactor issue?

**Pull request merged**